### DISCUSSION TITLE
Importer Not Working .Nothing Happens When Click On Start For Manual Execution

### DISCUSSION THREAD

**Discussion by Makskknd** [pimcore/pimcore#9539] [Q&A / Support]
Good day!
I am having a problem with the Pimcore Data Importer - it cannot load category and product data from an XML file.

When I click on "Start", the process does not go and the progress bar does not move, no errors are generated.

I (and not only me) wrote in more detail in this thread:
https://github.com/pimcore/data-importer/issues/41

Perhaps someone faced a similar problem and was able to solve it? I would be grateful if you can help.

https://user-images.githubusercontent.com/13482518/120851539-9f5c2000-c581-11eb-8c09-612ba8c6f417.png

**Comment by fashxp** [comment]
did you follow [my latest post](https://github.com/pimcore/data-importer/issues/41#issuecomment-865658975) in the issue? 

----- 
hmm, then the queue is empty and nothing is there to import. 

the import process consists of two steps: 
1) preparation: data source is read and queue items for import are created. 
2) processing: processing of the queue and actual importing of the data. 

step 1) this is executed when the import button is pressed. 
step 2) is executed by the commands I mentioned above (depending on import type there is sequential and parallel processing). 

For details please also have a look [in the docs](https://pimcore.com/docs/data-importer/current/Import_Execution_Details.html). 

your case (progress bar stands still, but execution of commands does not process any items) only should happen, if some error happened. steps to debug:
- Have a look at the `bundle_data_hub_data_importer_queue` table - is it empty, or are there some entries? 
- If there are entries, retry executing the processing jobs. Depending on the `jobType` column value, `datahub:data-importer:process-queue-parallel` or `datahub:data-importer:process-queue-sequential` is responsible for executing. 
- Are there any log entries in application logger or Pimcore log files? 

Please share your findings ...

**Comment by xalessandrox** [comment]
Hello! I join here so I don't open a new discussion on same topic.
I am not going through this whistling and with hand in my pocket.

In my case all objects are perfectly loaded in the queue like charm but they are only imported in pimcore via console with 
`datahub:data-importer:process-queue-parallel` or also sequentially. Nothing happens in Pimcore interface and the progress bar rimains at 0.  So if I don't execute the command in the terminal, no objects are imported.

I looked in the docs: Apparently I  need to activate the symfony messenger based with the following:
```
pimcore_data_importer:
    messenger_queue_processing:
        activated: true
```
If this is not wrong, my question is: Where do I have to put the setting?

Maybe is a simple question because I am new to Symfony but any help would be appreciated. Thank you!
